.github/workflows/build.yml

```diff
@@ -279,7 +279,7 @@ jobs:
           npm install
           cd ../tests
           npm install
-          node_modules/.bin/mocha --parallel -j 7 -r ts-node/register 'tests/**/test-*.ts'
+          TS_NODE_TRANSPILE_ONLY=true node_modules/.bin/mocha --parallel -j 7 -r ts-node/register 'tests/**/test-*.ts'
```

Comment:
We should not skip Transpilation, otherwise it will compile even if there are errors

Reply:
OK makes sens. It's strange because now (ever since I updated the dependencies) we have all these typing errors related to the api returning `unkown` type when it would return a generated type before